data/korath outfits.txt

```diff
@@ -310,3 +310,12 @@ outfit "Steering (Stellar Class)"
 	"turning energy" 8.2
 	"turning heat" 26.7
 	description "After millennia of war, the Korath have developed some truly enormous warships. These steering systems give even their largest warships a reasonable amount of agility in combat."
+	
+outfit "Korath Repeater Rifle"
+	category "Hand to Hand"
+	cost 220000
```

Comment:
220,000 credits for a single rifle seems a little ridiculous--especially considering that you can't buy these, so players will be able to sell them for quite a pretty penny.